js/ui/windowManager.js

```diff
@@ -1289,6 +1289,11 @@ const WindowManager = new Lang.Class({
             }
```

Comment:
To have a seamless experience the GNOME Builder icon should not be seen in the navigation bar nor should appear in alt-tab. I do keep track of the state though - hence when you switched to Builder and you focus another app, when you alt-tab back you will get to the Builder window. Though in the alt-tab widget the app icon will be visible. But yeah - Builder is basically just another mode of the app you "modify". That is at least how I am understanding it.